biocypher_metta/adapters/gene_ontology_adapter.py

```diff
@@ -0,0 +1,116 @@
+import rdflib
+from owlready2 import *
+from biocypher_metta.adapters.ontologies_adapter import OntologyAdapter
+
+class GeneOntologyAdapter(OntologyAdapter):
+    def __init__(self, write_properties, add_provenance, type, label='ontology term', dry_run=False):
+        super().__init__(write_properties, add_provenance, label, type, dry_run)
+        self.source = "Gene Ontology"
+        self.source_url = "http://purl.obolibrary.org/obo/go.owl"
+        self.ONTOLOGIES = {
+            'go': 'http://purl.obolibrary.org/obo/go.owl'
+        }
+
+
+    def get_graph(self, ontology='go'):
```

Comment:
This method already exists in the parent class and the one in the parent class should be used

Reply:
I still don't see a reason for this method to exist here. can't you call `self.graph(ontology='go')` when you want the `graph` method in the parent class